os/net/blemgr/bledev.c

```diff
@@ -50,11 +99,23 @@ int bledev_handle(struct bledev *dev, lwnl_req cmd, void *data, uint32_t data_le
 		}
 
 		TRBLE_DRV_CALL(ret, dev, init, (dev, t_client, t_server));
+
+		if (ret == TRBLE_SUCCESS) {
+			sem_init(&g_scan_countsem, 0, 0);
+			int tid = task_create("bledev_handler", 103, 2048, (main_t)_bledev_handler, NULL);
```

Comment:
Please use `kernel_thread()` instead of `task_create`
`task_create` is for user side API, `kernel_thread()` is as same as `task_create`, but it is for kernel side.

Reply:
I changed it